pkgs/by-name/do/docuseal/Gemfile

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+ruby '3.3.4'
```

Comment:
This is too old for the new nixpkgs (try rebasing your PR branch on master)

Reply:
i can confirm it failing... But sadly I do not know a good fix for it. just remove this line?